jest/setup.js

```diff
@@ -129,6 +129,13 @@ const mockNativeModules = {
     addListener: jest.fn(),
     removeListeners: jest.fn(),
   },
+  Linking: {
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
```

Comment:
I'm confused, these methods are not exposed in the `Linking` library:
`Libraries/Linking/Linking.js:127`